package.json

```diff
@@ -6,7 +6,7 @@
   "scripts": {
     "bundle": "webpack --progress --colors",
     "dev": "webpack --progress --colors --watch",
-    "test": "eslint . && istanbul cover --include-all-sources babel-node node_modules/.bin/jasmine"
+    "test": "istanbul cover --include-all-sources babel-node node_modules/.bin/jasmine && eslint ."
```

Comment:
Because it was failing and never running the tests if I had a linting error (such as console.log)